gravitee-apim-console-webui/src/services-ngx/api-logs-v2.service.spec.ts

```diff
@@ -50,6 +50,7 @@ describe('ApiLogsV2Service', () => {
       { queryParams: { applicationIds: '1,2' }, queryURL: '?page=1&perPage=10&applicationIds=1,2' },
       { queryParams: { planIds: '1,2' }, queryURL: '?page=1&perPage=10&planIds=1,2' },
       { queryParams: { applicationIds: '1,2', planIds: '1,2' }, queryURL: '?page=1&perPage=10&applicationIds=1,2&planIds=1,2' },
+      { queryParams: { methods: '1,2' }, queryURL: '?page=1&perPage=10&methods=1,2' },
```

Comment:
just to be explicit it would be better to use real values. wdyt ?
```suggestion
      { queryParams: { methods: '1,2' }, queryURL: '?page=1&perPage=10&methods=GET,POST' },
```